internal/crossdock/server/oneway/server.go

```diff
@@ -22,21 +22,36 @@ package oneway
 
 import (
 	"fmt"
+	"os"
+	"time"
 
 	"go.uber.org/yarpc"
 	"go.uber.org/yarpc/encoding/json"
 	"go.uber.org/yarpc/encoding/raw"
 	"go.uber.org/yarpc/internal/crossdock/thrift/oneway/yarpc/onewayserver"
 	"go.uber.org/yarpc/transport"
 	"go.uber.org/yarpc/transport/http"
+	"go.uber.org/yarpc/transport/x/redis"
 )
 
 var dispatcher yarpc.Dispatcher
 
+const redisAddr = "redis:6379"
+
 // Start starts the test server that clients will make requests to
 func Start() {
 	inbounds := []transport.Inbound{http.NewInbound(":8084")}
 
+	if redisIsAvailable() {
+		rds := redis.NewInbound(
+			redis.NewRedis5Client(redisAddr),
```

Comment:
Suggest moving the addr inline. These tests are used by customers as docs, having the addr inline makes it easier to understand (I don't have to deref the const in my mind).